packages/dialtone-vue3/components/icon/icon.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <component
     :is="icon"
-    v-show="name"
+    v-if="icon"
```

Comment:
Why it wasn't the same as the vue2 here?

Reply:
Maybe due to some rendering differences on vue 2 and vue 3 trying to import the SVG's directly, but that shouldn't be a problem anymore